LSP-volar.sublime-settings

```diff
@@ -50,6 +60,15 @@
 			// ST4
 			"document_selector": "text.html.vue",
 			"feature_selector": "text.html.vue",
-		}
+		},
+		{
+			"languageId": "typescript",
+			// ST3
+			"scopes": ["source.ts"],
+			"syntaxes": ["Packages/TypeScript Syntax/TypeScript.tmLanguage"],
+			// ST4
+			"document_selector": "source.ts",
+			"feature_selector": "source.ts",
+		},
```

Comment:
What about JS? Is it not designed to work with those?

Reply:
Volar does seem to work, for javascript files. as long as there is a jsconfig.json or tsconfig with allowJs and checkJs enabled.

Reply:
So like LSP-typescript. We should probably enable it for JS too then.

Reply:
Not sure why volar is not picking up types from `types` array in `tsconfig.json` for me. Neither for JS files, nor Vue files.

Reply:
Can you share your tsconf.json file?

Reply:
Not for that specific project but I might try to reproduce it later on some simpler project.

BTW. See https://github.com/johnsoncodehk/volar/blob/bd2a7bad02deb2e0b04156ca1b788e2cb6c934ce/packages/client/src/common.ts#L97-L104

It looks like we should also include the JSX, TSX scopes here.

Reply:
Added tsx and jsx.